Gemfile

```diff
@@ -64,3 +64,4 @@ group :docs do
 end
 
 gemspec path: "."
+gem "arbre", github: "activeadmin/arbre", branch: "fix-keyword-delegation-in-form-component"
```

Comment:
@javierjulio  thanks, are we going to merge it or first we will update arbre ?

Reply:
I think we can merge and release arbre, and then update this?

Reply:
@deivid-rodriguez  mee too

Reply:
Thanks guys! I will clean up the test cases I added before merging. If we release the Arbre gem before then, feel free to update here or I can do that.

Reply:
Ok, all done updating tests so this is ready.